app/assets/stylesheets/local/responses/_index.scss

```diff
@@ -17,22 +17,6 @@
       color: $success-green;
     }
   }
-
-  // Search excerpt
-  tr.second_row td {
```

Comment:
`second_row` is used in `app/views/layouts/_index_table.html.erb` but am I correct in thinking this style isn't used there because it's local to `responses`? Hard for me to tell.

Reply:
Let's leave this in case we bring excerpts back.